.gitignore

```diff
@@ -30,3 +30,4 @@ CMakeLists.txt.user
 *swp
 .orig
 .vagrant
+.*.Rnb.cached
```

Comment:
Did you mean to include this in the changelist? ATM it's added automatically by RStudio but it's going away soon (@jjallaire and I decided that we're going to bury these cache folders in `.Rproj.user`).